cypress/e2e/internal/billing/supplementary/replace-cv-2023-fin-year-no-changes.cy.js

```diff
@@ -0,0 +1,370 @@
+    // choose Yes, approve and continue
+    cy.get('.govuk-radios > :nth-child(1) > #reviewOutcome').click()
+    cy.get('.govuk-button').contains('Continue').click()
+
+    // click the Bill runs menu link
+    cy.get('#navbar-bill-runs').contains('Bill runs').click()
+
+    // Bill runs
+    // click the Create a bill run button
+    cy.get('#main-content > a.govuk-button').contains('Create a bill run').click()
+
+    // Which kind of bill run do you want to create?
+    // choose Supplementary and continue
+    cy.get('input#selectedBillingType-2').click()
+    cy.get('form > .govuk-button').contains('Continue').click()
+
+    // Select the region
+    // choose Test Region and continue
+    cy.get('input#selectedBillingRegion-9').click()
+    cy.get('form > .govuk-button').contains('Continue').click()
+
+    // click the Bill runs menu link
+    cy.get('#navbar-bill-runs').contains('Bill runs').click()
+
+    // -------------------------------------------------------------------------
+    cy.log('Confirming and sending the SROC supplementary bill run')
+
+    // Bill runs
+    // we immediately select the SROC bill run. We don't expect it to be finished and to hit the spinner page but it
+    // might be super quick and already done. So we do no checks at this point
+    cy.get(':nth-child(1) > :nth-child(1) > .govuk-link').click()
+
+    // Test Region supplementary bill run
+    // we have to wait till the bill run has finished generating. The thing we wait on is the READY label. Once that
+    // is present we can confirm the bill run is as expected
+    cy.get('#main-content > div:nth-child(1) > div > p > strong', { timeout: 20000 }).should('contain.text', 'Ready')
+
+    // check the details and then click Confirm bill run
+    cy.get('dl').within(() => {
+      // date created
+      cy.get('@formattedCurrentDate').then((formattedCurrentDate) => {
+        cy.get('div:nth-child(1) > dd').should('contain.text', formattedCurrentDate)
+      })
+      // region
+      cy.get('div:nth-child(2) > dd').should('contain.text', 'Test Region')
+      // bill run type
+      cy.get('div:nth-child(3) > dd').should('contain.text', 'Supplementary')
+      // charge scheme
+      cy.get('div:nth-child(4) > dd').should('contain.text', 'Current')
+    })
+    cy.get(':nth-child(2) > .govuk-grid-column-two-thirds').should('contain.text', '£0.00')
+    cy.get('.govuk-heading-l').should('contain.text', '1 supplementary bill')
+    cy.get('.govuk-table__body > .govuk-table__row > :nth-child(1)').should('contain.text', 'S00000007A')
+    cy.get('.govuk-table__body > .govuk-table__row > :nth-child(2)').should('contain.text', 'Big Farm Co Ltd 02')
+    cy.get('.govuk-table__body > .govuk-table__row > :nth-child(3)').should('contain.text', 'AT/SROC/SUPB/02')
+    cy.currentFinancialYearDate().then((result) => {
+      cy.get('.govuk-table__body > .govuk-table__row > :nth-child(4)').should('contain.text', '2023')
+    })
```

Comment:
We're not using `result` here so we can get rid of the call to `currentFinancialYearDate()`.
```suggestion
    cy.get('.govuk-table__body > .govuk-table__row > :nth-child(4)').should('contain.text', '2023')
```

Reply:
Well spotted, I'll sort that out